lib/tools.js

```diff
@@ -294,7 +294,7 @@ function createArgsNormalizer (defaultOptions) {
     }
     if (prettyPrint) {
       const prettyOpts = Object.assign({ messageKey }, prettyPrint)
-      stream = getPrettyStream(prettyOpts, prettifier, stream)
+      stream = getPrettyStream.call(this, prettyOpts, prettifier, stream)
```

Comment:
`createArgsNormalizer` and `getPrettyStream` are only used internally, we can change the function signature without breaking the API.  Therefore pass in the `instance` as another parameter to `createArgsNormalizer` and instead of using call here pass the `instance` directly into the `getPrettyStream` function